### ISSUE TITLE
Add explicit support for multiple instances

### ISSUE THREAD

**Issue by rworsnop** [rworsnop/vertx-beans#1]
Would be good to be able to inject an object (InstanceRunner?) into the application and use it to execute code on multiple instances. Useage of the RxJava version could look something like this:

```
// Run code in 4 instances:
instanceRunner.execute(4, () ->{
  return vertx.createHttpServer(options).requestHandler(handler).listenObservable();
}).subscribe(empty(), this::handleError, this::handleComplete;
```

This would create 4 HTTP servers, each listening on its own event loop. We can use Observable.merge to combine them into a single Observable to return from execute()

**Comment by markhofmann** [comment]
+1 - is anybody working on that?

**Comment by rworsnop** [comment]
Yeah, will do, now that I know someone else wants it as well.

**Comment by markhofmann** [comment]
Perfect! Btw: I'm using your project together with vertx-nubes. The combination is really great as a spring-web replacement.

**Comment by rworsnop** [comment]
Done. Just released 1.1.0.
I've added some stuff to the README and api docs are here: http://rworsnop.github.io/vertx-beans/apidocs/
